src/classes_def.xml

```diff
@@ -1,7 +1,9 @@
 <lcgdict>
   <class name="TTAnalysis::BaseObject"/> 
   <class name="std::vector<TTAnalysis::BaseObject>"/>
-  <class name="TTAnalysis::Lepton"/>
+  <class name="TTAnalysis::Lepton">
+    <field name="hlt_already_matched" transient="true"/>
```

Comment:
This flag is only used internally to check if we already tried to match this lepton to an online object, so we can avoid redoing the same thing if the lepton is in more than 1 dilepton pair. So we don't need this info in the final tree.